bin/Model.js

```diff
@@ -37,7 +37,7 @@ module.exports = class Model {
         this.weapons.push({ name: newMatch.weaponName, number: newNumber });
     }
 
-    handleSelectionDataRecursive (selectionData) {
+    handleSelectionDataRecursive (selectionData, number) {

```

Comment:
ditto, maybe the same name as above, in order to keep the same name across all the code base